specs/waku/waku-1.md

```diff
@@ -170,9 +170,13 @@ packet-format = "[" packet-code packet-format "]"
 
 required-packet = 0 status /
                   1 messages /
-		  22 status-update /
+                  22 status-update /
 
-optional-packet = 126 p2p-request / 127 p2p-message
+optional-packet = 11 batch-ack /
+                  12 message-response /
+                  126 p2p-request-complete /
```

Comment:
This is not ideal as the content of the packet is not an evenlope but directly an rlp list. This is then only defined in the mailserver specification.